### PR TITLE
feat: province deaths pipeline

### PR DESCRIPTION
**Pull request by noppakorn**
- Generate province deaths data from djay/covidthailand

### CODE REVIEW

File: bot_jobs/build_province_deaths.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import pandas as pd
+import datetime
+import json
+
+DEATH_DATA_14DAYS_OUT_PATH = "../components/gis/data/province-deaths-data-14days.json"
```
Comment (porames) [inline bot_jobs/build_province_deaths.py]: I think we should combine it to province-data-14days.json

File: .github/workflows/covid-scheduled-bot.yaml [hunk 2]
```diff
@@ -32,6 +32,9 @@ jobs:
             - name: Get National Data From djay/covidthailand
               working-directory: bot_jobs
               run: python get_national_data.py
+            - name: Get and Build Province deaths
```
Comment (porames) [inline .github/workflows/covid-scheduled-bot.yaml]: Add death data to build_job.json generated by build_province_graph.py so all fields in the table can use one file

File: bot_jobs/build_province_deaths.py (hunk 1 above)
Comment (noppakorn) [reply bot_jobs/build_province_deaths.py]: Combined in latest commit, code maybe a bit messy.

File: .github/workflows/covid-scheduled-bot.yaml (hunk 2 above)
Comment (noppakorn) [reply .github/workflows/covid-scheduled-bot.yaml]: Resolved

**Pull request merged**